packages/database/lib/modular/index.d.ts

```diff
@@ -0,0 +1,217 @@
+import { ReactNativeFirebase } from '@react-native-firebase/app';
+import { FirebaseDatabaseTypes } from '..';
+
+import FirebaseApp = ReactNativeFirebase.FirebaseApp;
+import Database = FirebaseDatabaseTypes.Module;
+import DatabaseReference = FirebaseDatabaseTypes.Reference;
+
```

Comment:
missing `getServerTime()` & `useEmulator()`

Reply:
`useEmulator()` has been renamed to `connectDatabaseEmulator()`. `getServerTime()` actually doesn't exist in the V9 API. I think that was left over from when I was copying over some functions from V8 (compat)